app/js/components/JobList.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import JobRow from './JobRow';
+
+const JobList = ({ activeJobs }) => (
```

Comment:
Could we add a "failed" jobs section which has the stack-trace and a remove button? This would make things much easier during development.